BuildMonitor/Domain/ISolutionBuild.cs

```diff
@@ -6,6 +6,8 @@ public interface ISolutionBuild : ITimer, IPersistable
     {
         DateTime Started { get; }
         ISolution Solution { get;  }
+        bool IsRebuildAll { get; set; } // bit of a shame to make this mutable
```

Comment:
This is because we don't know it when we construct the object, but I'm happy to remove the comment.